ISSUE_TEMPLATE.md

```diff
@@ -3,3 +3,30 @@ Please do not submit feature requests. The [Community Forums][1] has a
 section for submitting, voting for, and discussing product feature requests.
 [1]: https://community.bitwarden.com
 -->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Steps To Reproduce**
+How can we reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. Click on '...'
+
+**Expected Result**
+A clear and concise description of what you expected to happen.
+
+**Actual Result**
+A clear and concise description of what is happening.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Mobile**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - App Version [e.g. 2.3.0]
```

Comment:
It would help if users specifically stated if they're using a beta release (it's technically possible to push updates to the beta channel with the same version string (on Android anyway))

Reply:
Also add "Build Version". A user can get this under Settings > About in the app.

Reply:
Build version is basically what I meant by "App Version" unless those are 2 different things for us?

Reply:
Yes, they are. Check out Settings > About in the app.